content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

```diff
@@ -123,6 +123,14 @@ $ kubectl edit clusterrole system:kube-scheduler
     - update
 ```
 
+my-scheduler runs as the `default` service account in the `kube-system` namespace. To allow it run with super-user access, grant cluster-admin permissions to the `default` service account in the `kube-system` namespace.
+
+``` shell
+kubectl create clusterrolebinding my-scheduler-cluster-admin \
+  --clusterrole=cluster-admin \
+  --serviceaccount=kube-system:default
```

Comment:
curious why we are not using the `system:kube-scheduler` clusterole?

Reply:
In general, you should not give elevated privileges of cluster-admin to default unless absolutely needed. Did you add my-scheduler to resourceNames of kube-scheduler clusterrole?

Reply:
Yes, added. And I also agree that binding `cluster-admin` is not a good idea.

Changed to `system:kube-scheduler`